server/app/services/mongo_pubsub.rb

```diff
@@ -158,14 +159,8 @@ def tail!
         self.collection.find(query, {cursor_type: :tailable_await, batch_size: 100}).sort(:$natural => 1).each do |item|
           channel = item['channel']
           data = item['data']
-          
-          subscribers = self.subscriptions.select{|s| s.channel == channel }
-          subscribers.each do |subscription|
-            begin
-              subscription.queue_message(data.dup) if subscription
-            rescue Celluloid::DeadActorError
-            end
-          end
+
+          actor.async.queue_message(channel, data)
```

Comment:
Still sneaking this in, because I forgot to `git push` it yesterday.

Thinking this is better for perf, and it shouldn't violate ordering.